vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -401,22 +403,14 @@ func (r *RefCountsMap) discoverAndSync(c *client.Client, d drivers.VolumeDriver)
 			if isVMDKMount(mount.Source) != true {
 				continue
 			}
-			volname := mount.Name
-			// gets hit once to retrieve the default datastore. datastoreName is reused henceforth
-			if datastoreName == "" {
-				datastoreName, _, err = plugin_utils.GetDatastore(volname, d)
```

Comment:
this isn't needed anymore. returned datastore as part of volumeInfo. captured here to be reused further (and avoid trip to esx)